base/certificates.yaml

```diff
@@ -0,0 +1,56 @@
+# https://cert-manager.io/docs/configuration/selfsigned/#bootstrapping-ca-issuers
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: cockroachdb-ca
+spec:
+  isCA: true
+  commonName: cockroachdb-ca
+  dnsNames:
+    - cockroachdb-ca
+  secretName: cockroachdb-ca
+  issuerRef:
+    name: selfsigned-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: ca-issuer
```

Comment:
This should probably have a name related to Cockroach, so that its not used for other Certificates in the namespace by accident?